tests/schemas/test_openapi.py

```diff
@@ -52,6 +53,15 @@ def test_list_field_mapping(self):
             with self.subTest(field=field):
                 assert inspector._map_field(field) == mapping
 
+    def test_lazy_string_field(self):
+        class Serializer(serializers.Serializer):
+            text = serializers.CharField(help_text=_('lazy string'))
```

Comment:
I think we can do this without the serialiser even... 

```
mapping = inspector._map_field(serializers.CharField(help_text=_('lazy string')))
assert isinstance(mapping['description'], str), "description must be str"
```

or similar...

Reply:
`_map_fields()` doesn't create the description, that happens in `_map_serializer()`, so we must check it there, I believe.

Reply:
Good, yes, you're right. Super then! 👍